Yona/Yona/DataService/UserManager.swift

```diff
@@ -50,6 +50,10 @@ class UserManager: NSObject {
                         self.userInfo = dict
                         #if DEBUG
                         if let code = dict["mobileNumberConfirmationCode"] { print(">>>>>>>>>>>> SMS Confimation code: " + "\(code)" + "<<<<<<<<<<<<<") }
+                            
+                            let alert = UIAlertView(title:code, message:nil, delegate:nil, cancelButtonTitle:"OK")
```

Comment:
Calling alert popup from usermanager will crash...can't do this here, although Ic annot see this code in the branch not sure why it is in PR